### PR TITLE
3.5.2 - 14053 - canceling welcome wizard closes app unless show-wizard-at-sta…

### PR DESCRIPTION
**Pull request by Cattlesquat**
…rtup is unchecked

### CODE REVIEW

File: vassal-app/src/main/java/VASSAL/build/module/WizardSupport.java [hunk 1]
```diff
@@ -188,7 +188,18 @@ protected Void doInBackground() {
...
+        if (g.shutDown()) {
+          System.exit(0);
+        }
+        else {
+          g.getPlayerWindow().setVisible(true);
+        }
+      }
+      else {
+        g.getPlayerWindow().setVisible(true);
+      }
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/build/module/WizardSupport.java:202]: Couldn't you eliminate both `else` cases and stick `g.getPlayerWindow().setVisible(true);` below the end of the outer `if`?

Comment (uckelman) [reply vassal-app/src/main/java/VASSAL/build/module/WizardSupport.java:196]: In fact, just have one `if`: `if (Boolean.TRUE.equals(showWizard) && g.shutDown())`

Comment (Cattlesquat) [reply vassal-app/src/main/java/VASSAL/build/module/WizardSupport.java:196]: done

Comment (Cattlesquat) [reply vassal-app/src/main/java/VASSAL/build/module/WizardSupport.java:194]: Alzo Alzo done

**Comment by uckelman** [comment]
@BrentEaston Thoughts on this change?

**Comment by BrentEaston** [comment]
I have no problem with that.

**Pull request merged**